es6-destructuring/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <script>
+      const book1 = {
+        title: 'Goodnight Punpun',
+        author: 'Inio Asano',
+        libraryID: 3353,
+      };
+      const { title, author, libraryID } = book1;
+      console.log(
+        'The title of the book is ' +
+          title +
+          ', ' +
+          'the author is ' +
+          author +
+          ', ' +
+          'and the library id is ' +
+          libraryID +
+          '.'
+      );
+
+      const book2 = {
+        title: 'Les Fleurs du mal',
+        author: 'Charles Baudelaire',
+        libraryID: 2345,
+      };
+      const {
+        otherTitle = book2.title,
+        otherAuthor = book2.author,
+        otherLibraryID = book2.libraryID,
+      } = book2;
```

Comment:
You shouldn't be doing `book2.title`, `book2.author` etc. You should just be doing the normal property names on the left side of the colon, and the variable names on the right side of the colon.

`const { title: title2 } = book2`

Reply:
<img width="535" alt="Screenshot 2023-06-19 at 2 43 59 PM" src="https://github.com/Edward-J-Kim/c0523-code-solutions/assets/126144853/981a95e8-6b3c-4802-b4d4-d4caf20f1bb2">
updated